ci/docker/development/bitcoin/start.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -x
 btc_version=$(bitcoind --version)
-btc_block_time=5
+btc_block_time=15
```

Comment:
will this also affect localnets (and therefore maybe the bouncer)?

Reply:
Might be, lets see

Reply:
Yeah I believe it will - why do you need this change?

Reply:
I explained in the summary:
> In order to be able to thoroughly test our swapping app transitions, we need to increase the block producing time.

With boost, we have quite a few transitions that we need to make sure occur one after another and there are no overlaps. This is really hard to test on backspin right now but also pretty annoying on sisyphos with the waiting times as high as 30 minutes.

Reply:
Think we can probably just create a separate image for backspin - but might involve a little refactoring for parameterisation etc.

Reply:
yeah it would be great if we can use a different blocktime on backspin

Reply:
Just chatted with Martin, i am going to try to test things locally by stopping the btc chain from mining and manually creating the blocks one by one.

Reply:
So we can close this PR?

Reply:
it would still be nice to change this in backspin tbh. right now the blocktimes are too fast to test the boost flow

Reply:
Ok let's close this PR and create an issue then

Reply:
As Nik said, me trying this on localnet is going to unblock me at the moment but we still need the time configurable on backspin.

Reply:
> and create an issue then

?

Reply:
we don't need to keep this PR up if it's not going to be merged

Reply:
created this: https://linear.app/chainflip/issue/PLA-1301/separate-bitcoin-image-thats-used-in-backspin

So we can close this now?

Reply:
No, I just touched it up to have the desired behavior

Reply:
We already have this image. It's what the if/else statement does

Reply:
Ok yeah makes it simpler - not sure about overloading the "PRUNE" var though - would rather create a separate env var to avoid confusion.

Reply:
it's already overloaded to turn off pruning and start an electrum server. it should probably be renamed but idk if it's in the scope of this pr

Reply:
Don't think it's a good reason to make things worse, but sure, so we don't hold this up since it seems to be blocking

Reply:
will let Assem approve